src/routes/playground/tags/playground/tags/editor/tags/tabs/tabs.style.module.scss

```
@@ -16,6 +16,18 @@
   padding: 0.5rem 1rem;
   text-wrap: nowrap;
   border-right: 2px solid transparent;
+  input {
+    font: inherit;
+    color: inherit;
+    background: none;
+    border: none;
+    outline: none;
+    padding: 0;
+    // affordance for the editing state; box-shadow so the box stays the
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)